src/App.css

```diff
@@ -1,28 +1,45 @@
-.App {
-  text-align: center;
-}
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+* {
+  box-sizing: border-box;
 }
 
-.App-header {
-  background-color: #222;
-  height: 150px;
-  padding: 20px;
-  color: white;
+.board {
+  background: #a0e;
+  display: grid;
+  grid-gap: .5rem;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 300px));
+  grid-template-rows:    repeat(auto-fill, minmax(250px, 300px));
 }
 
-.App-title {
+header {
+  text-align: center;
+  padding: 10px;
   font-size: 1.5em;
 }
 
-.App-intro {
-  font-size: large;
+/* CSS Button */
+
+*{padding:0;margin:0;}
+
+body{
+	font-family:Verdana, Geneva, sans-serif;
+	font-size:18px;
+	background-color:#CCC;
+}
+
+.float{
+	position:fixed;
+	width:60px;
+	height:60px;
+	bottom:40px;
+	right:40px;
+	background-color:#0C9;
+	color:#FFF;
+	border-radius:50px;
```

Comment:
There's a weird-looking sliver of a white border around this button-- it's caused because the default border added (via border-radius) is 1px. It's probably best to add `border-width: 0;`